tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+
+[testenv:unit]
+
+deps =
+    pytest
+
+recreate =
+    false
+
+commands =
+    pip install -e .
+    pytest -vv tests/unit_tests.py
+
+[testenv:contrib]
+
+deps =
+    pytest
+
+recreate = false
+
+commands =
+    pip install -e .
+    pytest -vv tests/contrib_end_to_end_test.py
```

Comment:
I think the coverage part (`--cov=mplc` & `--cov=mplc --cov-append`) is missing here and in line 36 or it should be added only to the CI yaml!